sklearn/tree/tests/test_tree.py

```diff
@@ -1547,10 +1547,11 @@ def check_raise_error_on_1d_input(name):
     assert_raises(ValueError, est.predict, [X])
 
 
-@ignore_warnings
-def test_1d_input():
-    for name in ALL_TREES:
-        yield check_raise_error_on_1d_input, name
+# XXX
```

Comment:
What's happening here for the XXX?

Reply:
It was probably a note in the making, don't remember now - possibly related to https://github.com/scikit-learn/scikit-learn/issues/11109 about incompatibility of `ignore_warnings` with parametrization. Thanks for catching this.

Removed it, and used  `ignore_warnings` as a context manager. Hopefully this will be fixed in #11109